src/gps/GPS.cpp

```diff
@@ -502,11 +503,22 @@ void GPS::setGPSPower(bool on, bool standbyOnly, uint32_t sleepTime)
             }
             msglen = gps->makeUBXPacket(0x02, 0x41, 0x08, gps->_message_PMREQ);
             gps->_serial_gps->write(gps->UBXscratch, msglen);
+        } else if (gnssModel == GNSS_MODEL_MTK) {
+            // Enter soft standby mode
+            LOG_DEBUG("L76K entering soft standby mode\n");
+            gps->_serial_gps->write("$PMTK161,0*28\r\n");
+            gps->_serial_gps->flush();
+            delay(250);
         }
     } else {
         if (gnssModel == GNSS_MODEL_UBLOX) {
             gps->_serial_gps->write(0xFF);
             clearBuffer(); // This often returns old data, so drop it
+        } else if (gnssModel == GNSS_MODEL_MTK) {
+            LOG_DEBUG("L76K exit soft standby mode\n");
+            // Start vehicle mode (to exit standby mode)
+            gps->_serial_gps->write("$PCAS11,3*1E\r\n");
+            delay(250);
         }
```

Comment:
if the flush() call resolves the problem it's needed here too

Reply:
I added that for good measure when I was troubleshooting why the power consumption wasn't lowering. I can probably yank it honestly.